src/main/java/core/mate/academy/service/BuildozerProducer.java

```diff
@@ -0,0 +1,22 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BuildozerProducer implements MachineProducer<Bulldozer> {
+    private final String[] bulldozerNames = {"John Deere", "CAT"};
+    private final String[] bulldozerColors = {"Yellow", "Orange"};
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        for (int i = 0; i < bulldozerNames.length; i++) {
+            Bulldozer bulldozer = new Bulldozer();
+            bulldozer.setName(bulldozerNames[i]);
+            bulldozer.setColor(bulldozerColors[i]);
+            bulldozers.add(bulldozer);
+        }
+        return List.of(bulldozers.toArray(new Bulldozer[0]));
+    }
+}
```

Comment:
Actually my previous comment was confusing :( Idk whether I used 'suggestion' wrongly or this function just worked incorrectly. So, let's repeat one more time :)
```suggestion
public class BuildozerProducer implements MachineProducer<Bulldozer> {
    @Override
    public List<Bulldozer> get() {
        return List.of(new Bulldozer(...), new Bulldozer(...));
    }
}
```
But I just noticed that you don't have an overridden constructor. In this case, your first solution is also applicable :)